### DISCUSSION TITLE
Allow importing custom default settings from settings UI

### DISCUSSION THREAD

**Discussion by arky** [digidem/mapeo-mobile#770] [Ideas]
Deploying customized icons/profiles aka. [default setting](https://github.com/digidem/mapeo-default-settings) requires physical access to target mobile phone using ADB tool chain and a skilled technical person on site to do this.  

Perhaps a simple solution would be import profile setting in Mapeo-mobile setting UI which enables the user to import customized default settings from the mobile device itself. The customized default settings could be sent to mobile device via a email or social media messaging.

**Comment by gmaclennan** [comment]
There is a settings screen in v2.1.0 #358 which allows you to import a `.mapeosettings` file without needing adb or moving files around. @arky would be great if you could test this and see if it solves this issue. We recognize that we need to further improve this process in the future (e.g. syncing settings between devices)

**Comment by arky** [comment]
@gmaclennan  Great, I'll do some testing with 2.1.0 release this weekend.
